src/Collection/ProcessArgumentsCollection.php

```diff
@@ -132,6 +132,15 @@ public function addOptionalBooleanArgument(string $argument, $value, string $tru
      * @param int|null $value
      */
     public function addOptionalIntegerArgument(string $argument, $value)
+    {
+        $this->addOptionalMixedArgument($argument, $value);
+    }
+
+    /**
+     * @param string $argument
+     * @param mixed $value
+     */
+    public function addOptionalMixedArgument(string $argument, $value)
```

Comment:
@veewee I think there is no need to add this new function? Can't we just use `addOptionalArgument()`?

Reply:
No it can't.
The `addOptionalArgument()` method does not add the argument on e.g. value `0`.
In this case `0` is a valid value, so we have to use a `null` check to make sure it falls back to the phsptan configuration file or phpstan default value instead.